src/core/positioning/nmeagnssreceiver.h

```diff
@@ -51,10 +51,38 @@ class NmeaGnssReceiver : public AbstractGnssReceiver
     void handleStartLogging() override;
     void handleStopLogging() override;
 
+    void handleIMUCorrection( const QString &sentence );
+
     QTime mLastGnssPositionUtcTime;
 
     QFile mLogFile;
     QTextStream mLogStream;
+
+    GnssPositionInformation mCurrentNmeaGnssPositionInformation;
+
+    struct IMUCorrectedPosition
```

Comment:
I think we don't need corrected here, it's implied anyways.

```suggestion
    struct IMUPosition
```